metaflow/plugins/kubernetes/kube_utils.py

```diff
@@ -23,3 +23,26 @@ def parse_cli_options(flow_name, run_id, user, my_runs, echo):
             raise CommandException("A previous run id was not found. Specify --run-id.")
 
     return flow_name, run_id, user
+
+
+def qos_requests_and_limits(qos: str, cpu: int, memory: int):
+    "return resource requests and limits for the kubernetes pod based on the given QoS Class"
+    # Determine the requests and limits to define chosen QoS class
+    qos_limits = {}
+    qos_requests = {}
+    if qos == "Guaranteed":
```

Comment:
maybe a case insensitive check?

Reply:
there's validation in the kubernetes decorator that is case-sensitive as well. I can also make all defaults lower case but `besteffort` reads a bit clunky.

Reply:
validation changed to case insensitive.